tests/datalab/test_datalab.py

```diff
@@ -1113,3 +1114,48 @@ def test_find_imbalance_issues_no_args(self, imbalance_labels):
         assert "class_imbalance" in summary["issue_type"].values
         class_imbalance_summary = lab.get_issue_summary("class_imbalance")
         assert class_imbalance_summary["num_issues"].values[0] > 0
+
+
+class TestSpuriousCorrelations:
+    @pytest.fixture
+    def lab(self):
+        # Load dataset, sample for speed
+        dataset = load_dataset("fashion_mnist", split="train").shuffle(seed=0).select(range(30))
```

Comment:
We can consider using the `streaming` option of `load_dataset` to avoid downloading the entire Fashion-MNIST Dataset.  E.g.

```
dataset = load_dataset("fashion_mnist", split="train", streaming=True).take(30)
```

Reply:
any unit test like this needs to be marked as slow

Reply:
should ideally replace this with a toy dataset. Do not need a real image dataset to test this basic code